plugins/meta/firewall/iptables.go

```diff
@@ -33,6 +34,7 @@ func getPrivChainRules(ip string) [][]string {
 }
 
 func ensureChain(ipt *iptables.IPTables, table, chain string) error {
```

Comment:
If we switch to the utils package, this function should be deprecated.

Reply:
Oops! Fixed.